kratos/test/expected.data.yml

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright 2021 SkyAPM org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+segmentItems:
+  {{- range .segmentItems }}
+    {{- if eq .serviceName "kratos-server" }}
+    - serviceName: kratos-server
+      segmentSize: {{ gt .segmentSize 0 }}
+      segments:
+      {{- range .segments }}
+      - segmentId: {{ notEmpty .segmentId }}
+        spans:
+        {{- range .spans}}
+        - operationName: /helloworld.Greeter/SayHello
+          operationId: 0
+          parentSpanId: -1
+          spanId: 0
+          spanLayer: RPCFramework
+          startTime: {{ gt .startTime 0 }}
+          endTime: {{ gt .endTime 0 }}
+          componentId: 5010
+          isError: false
+          spanType: Entry
+          peer: ''
+          skipAnalysis: false
+          {{- if eq (index .refs 0).networkAddress "kratosserver:8000"}}
+          tags:
+            - { key: User-Agent, value: Go-http-client/1.1 }
+          {{- end}}
+          refs:
+            - parentEndpoint: /helloworld.Greeter/SayHello
+              networkAddress: {{ notEmpty (index .refs 0).networkAddress }}
+              refType: CrossProcess
+              parentSpanId: 0
+              parentTraceSegmentId: {{ notEmpty (index .refs 0).parentTraceSegmentId }}
+              parentServiceInstance: {{ notEmpty (index .refs 0).parentServiceInstance }}
+              parentService: kratos-client
+              traceId: {{ notEmpty (index .refs 0).traceId }}
+        {{- end}}
+      {{- end}}
+    {{- end}}
+    {{- if eq .serviceName "kratos-client" }}
+    - serviceName: kratos-client
+      segmentSize: {{ gt .segmentSize 0 }}
+      segments:
+      {{- range .segments }}
+      - segmentId: {{ notEmpty .segmentId }}
+        spans:
+        {{- range .spans}}
+        - operationName: /helloworld.Greeter/SayHello
+          operationId: 0
+          parentSpanId: -1
+          spanId: 0
+          spanLayer: RPCFramework
+          startTime: {{ gt .startTime 0 }}
+          endTime: {{ gt .endTime 0 }}
+          componentId: 5011
+          isError: false
+          spanType: Exit
```

Comment:
Is `EntrySpan` missing ?

Reply:
For the client, entry request is using std http handler instead of the kratos rpc framework, so the client does't have `EntrySpan`

Reply:
I think @arugal asked for a full instructions. Even it is not a part of this plugin, it is better to activate plugin for it too.